index.markdown

```diff
@@ -13,52 +16,20 @@ layout: default
+In the Fall of 2022 a few women runners in Oakland started talking about how we could come together to create a women’s run group in the East Bay. Women’s Run Club East Bay evolved out of that! This club is still in a new stage and will be what we build together! This is a living website to share some thinking about who we are, how to join, and how we may grow in community.
 
 # Mission & Values
 Our mission is to foster a fun, accessible, come-as-you-are community for women and non-binary runners rooted in the East Bay. We want our club to uplift values of inclusion, local community engagement, & celebrating each other as athletes and people.
 
 # What
 A lot of what our club becomes will be what we build together, so some (many?!) things aren’t planned yet. For now we have a Slack space and we’ll aim to kick off some more formal programming in the new year. 
 
-# 2023 could look like:
-Free to join (we don’t have any costs yet). If you are in Slack, you are a member.
-Regularly scheduled weekday runs in Oakland (and hopefully some joint runs with other local groups!)
-Regularly scheduled weekend runs around the East Bay
-Active Slack forum for planning ad hoc meet-ups, discussing training, postpartum/injury recovery journeys, volunteering/community engagement opportunities, gear swap & more!
-Norming, forming, & storming as a club and maybe we can get some swag?!
-
-In the future we could incorporate as a non-profit if we think the structure will help us or charge a small membership fee if we have costs. 
-
-# FAQ
-## I have a friend who wants to join. How can I get them added to Slack? 
-Send them this form: [https://tinyurl.com/wrc-jointheteam](https://tinyurl.com/wrc-jointheteam). Our onboarding committee will commit to adding in new folks on a monthly basis.
-
-## Will this be competing with other East Bay run clubs? 
-There is not (that we know of!) an active women’s run club in the East Bay that has local roots, spans many running disciplines, and is free or low cost to join. In that sense, we hope this will be something new that will serve runners who we may not have met yet - runners craving similar paced teammates or looking for a women-centered space without a high annual price tag. For runners already plugged into an existing club, we hope that our group can be a value-add to the broader running community ecosystem - we are a “Yes and” if you will. Hopefully we can also have some joint runs with other East Bay running groups.
-
-## Is this a road specific run group? 
-We plan on scheduling road, track, and trail group runs. And there may be some members training for triathlons!
-
-## How do I find out about run meet ups?
-Join us on our Slack space!
-
-## Do I have to pay to join WRC? 
```

Comment:
I would still keep this. There is also no more mention of swag - we should still have that somewhere.